components/channel/include/channel_debug.h

```diff
@@ -0,0 +1,133 @@
+#ifndef CHANNEL_DEBUG_H_
+#define CHANNEL_DEBUG_H_
+
+#include "channel.h"
+
+#define LOG_LOCAL_LEVEL ESP_LOG_VERBOSE
+#include "esp_log.h"
+#include <string.h>
+
+/**
+ * channel_debug_print - print channel attributes
+ * @prefix: prefix string in debug output
+ * @ch: channel object to print
+ */ 
+static
+void
+__attribute__((unused))
+channel_debug_print
+(const char *prefix, const Channel *ch)
+{
+    ESP_LOGD(prefix, "@%p identifier:%s",
+             ch, ch->identifier);
+    ESP_LOGD(prefix, "@%p same.next:%p",
+             ch, list_entry(ch->same.next, struct channel, same));
+    ESP_LOGD(prefix, "@%p same.prev:%p",
+             ch, list_entry(ch->same.prev, struct channel, same));
+    ESP_LOGD(prefix, "@%p uniq.next:%p",
+             ch, list_entry(ch->unique.next, struct channel, unique));
+    ESP_LOGD(prefix, "@%p uniq.prev:%p",
+             ch, list_entry(ch->unique.prev, struct channel, unique));
+}
+
+/**
+ * channel_debug_print_dot - print channel attributes in dot format
+ * @prefix: prefix string in debug output
+ * @ch: channel object to print
+ */
+static
+void
+__attribute__((unused))
+channel_debug_print_dot
+(const char *prefix, const Channel *ch)
+{
+    if (strcmp("",ch->identifier)==0) {
```

Comment:
add spaces around `==`
```suggestion
    if (strcmp("",ch->identifier) == 0) {
```